radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenBalanceState.java

```diff
@@ -50,10 +50,11 @@ public BigDecimal getGranularity() {
 			return TokenClassReference.subunitsToUnits(granularity);
 		}
 
-		public Stream<OwnedTokensParticle> unconsumedConsumables() {
+		public Stream<OwnedTokensParticle> unconsumedTransferrable() {
 			return consumables.entrySet().stream()
 				.map(Entry::getValue)
 				.filter(c -> c.getSpin() == Spin.UP)
+				.filter(c -> !c.getParticle().getType().equals(FungibleType.BURNED))
```

Comment:
Would have a preference for `CONSTANT.equals(somevariable)` rather than `somevariable.equals(CONSTANT)`.